it/_rwd.md

```diff
@@ -31,13 +27,9 @@ Una volta che avrete dato il nome che preferite ai vostri _breakpoint_, avrete b
 
 <div class="note">
   <p>Ovviamente questo è un modo fin troppo semplice di gestire i <em>breakpoint</em>. Se avete bisogno di un metodo più permissivo, vi raccomando di non reinventare la ruota ed utilizzare qualcosa di testato ed efficace come: <a href="https://github.com/sass-mq/sass-mq">Sass-MQ</a>, <a href="http://breakpoint-sass.com/">Breakpoint</a> o <a href="https://github.com/eduardoboucas/include-media">include-media</a>.</p>
+  <p>If you are looking to read more on how to approach Media Queries in Sass, both <a href="http://www.sitepoint.com/managing-responsive-breakpoints-sass/">SitePoint</a> (from yours, truly) and <a href="http://css-tricks.com/approaches-media-queries-sass/">CSS-Tricks</a> have nice articles on this.</p>
```

Comment:
This sentence was missing. I added it in English, but it would need to be translated. :)